framework/src/wifi_manager/wifi_manager_lwnl_listener.c

```diff
@@ -165,10 +165,10 @@ int lwnl_fetch_event(int fd, void *buf, int buflen)
 	if (status.type == LWNL_DEV_WIFI) {
 		(void)_lwnl_call_event(fd, status, len);
 	} else {
-		char *buf = (char *)malloc(len);
-		int res = read(fd, buf, len);
-		NET_LOGI(TAG, "read unused data in the socket[%d]\n", res);
-		free(buf);
+		char *t_buf = (char *)malloc(len);
+		nbytes = read(fd, t_buf, len);
+		NET_LOGI(TAG, "read unused data in the socket[%d]\n", nbytes);
+		free(t_buf);
```

Comment:
In `ble_manager_lwnl_listener.c`, there is return after `free`. But in here, there is no.
Is it okay?

Reply:
I fixed it